fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -990,10 +990,12 @@ private void analyzeAddPartition(OlapTable olapTable, List<PartitionDesc> partit
         }
     }
 
-    private int calAvgBucketNumOfRecentPartitions(OlapTable olapTable, int recentPartitionNum) {
+    private int calAvgBucketNumOfRecentPartitions(OlapTable olapTable, int recentPartitionNum,
```

Comment:
how about move to `CatalogUtils` like `calBucketNumAccordingToBackends`?

Reply:
done